languages/common-lisp/reference/exercise-concepts/anagram.md

```diff
@@ -0,0 +1,20 @@
+# Concepts of Anagram
+
+While there are, of course, different ways to attack this problem the
+'canonical' way is to write a function that "normalizes" the
+candidates and the input word so that they can then be easily
+compared. It is this method which was reviewed for concepts. (ref. the
+[exercism v2 common-lisp track exercise][v2-exercise].)
+
+## Concepts
+
+- Sequences
```

Comment:
We might want to break this up a little? Into Strings, Lists, Arrays, Vectors, etc. Or if you are referring to the functions that they all share, maybe it should be Sequence Functions? This is just me nit-picking though.

Reply:
I was referring to the functions they all share. But also... the concept in Common Lisp is Sequences, or perhaps Sequential Data Types. Arrays and Lists (Conses) are sub-concepts and vectors and strings are sub-sub and sub-sub-sub concepts... 

But perhaps I am hewing to the type hierarchy too closely...

Maybe our track as a Sequences concept which is a "super-" concept which would include those concepts contained in other tracks such as array, vector & string? Or am I just being too pedantic and going taxonomy crazy? :)

Reply:
Looking to the Hyperspec and its chapters I see that Arrays, Strings and Sequences are separate. Sequences are about the functions they have in common. 

I'll go with 'Sequence Functions' for now until we find a better name.

Reply:
Okay, I think that seems reasonable. And yeah, the inheritance seems to get a little messy, but I think that Sequences as a super-topic (with the shared functionality) then the specific sub-types would work nicely.

I don't think it would be all too bad of an idea to introduce them in a hierarchy like that. It's good to have the relationships between those types straight and I think that teaching them in a tree like that would help the learner remember all the relationships.

Reply:
Should we use this PR to work out how sequences are taught, or defer that to a later PR?